src/components/God/index.tsx

```diff
@@ -1,12 +1,8 @@
 import React from 'react';
-import './SavedGod.scss';
+import './God.scss';
+import { GodFields } from '../../types';
 
-type Props = {
-  name: string;
-  thumbnail: string;
-};
-
-const SavedGod = ({ name, thumbnail }: Props) => {
+const God = ({ name, thumbnail }: GodFields) => {
   return (
     <>
       <div className="godImgAndName">
```

Comment:
i would rename this to be something like `godWrapper` rather than `godImgAndName`